.github/CODEOWNERS

```diff
@@ -0,0 +1,48 @@
+# Centrifudge Code Owners
+#
+# Order is important; the last matching pattern takes the most
+
+# These owners will be the default owners for everything in
+# the repo. Unless a later match takes precedence,
+*       @branan @mikiquantum @mustermeiszer @offerijns
```

Comment:
Btw, @offerijns usually doesn't have the bandwidth to respond to PRs in a short timeline. The other engineer in the Europe timezone is yourself, also not at full capacity. I would suggest adding myself here for that reason.

Reply:
Just sharing my two cents on this: I think it's important that we work on prioritising reviews above other work, and this is the right fix for that problem (and from what I've seen, this is going increasingly better in the protocol circle). But, as an example, if there is a PR that is touching the `pool-system` pallet, I think at least 1 of Branan/Frederik/me should review that (and the same logic applies to other pallets for other people, although `pool-system` is probably most critical). I don't think this should be reviewed solely by other people, it should imo always include someone from that list, even if this means waiting another day until they are available.

I'm wondering if this means we should not even remove this asterisk rule, and just have specific rules covering every individual folder. I checked the code owners docs, if a file is not covered by any of the rules (e.g. a change to just the README), then no code owner review would be required and any contributor with write access (i.e. the entire protocol circle) can review.

Reply:
My main reasoning is, that the asterix prevents us from missing a rule. I.e. We need to adapt the file everytime we add a new pallet and define a owner. 

But this is just another argument, I am not against removing it. Just to take into account.

Reply:
I think I see your points now @NunoAlexandre and @offerijns. I adapted the owner file.